docs/source/usage/generating_metaframes.rst

```diff
@@ -31,6 +31,8 @@ This function requires a :obj:`DataFrame` to be specified as parameter. The foll
 
     Internally, MetaSynth uses Polars (instead of Pandas) mainly because typing and the handling of non-existing data is more consistent. It is possible to supply a Pandas DataFrame instead of a Polars DataFrame to ``MetaDataset.from_dataframe``. However, this uses the automatic Polars conversion functionality, which for some edge cases result in problems. Therefore, we advise users to create Polars DataFrames. The resulting synthetic dataset is always a Polars DataFrame, but this can be easily converted back to a Pandas DataFrame by using ``df_pandas = df_polars.to_pandas()``.
 
+.. _OptionalParams:
```

Comment:
Isn't it possible to directly refer to the Optional Parameters section?

Reply:
Unfortunately not, this is the only way to do internal links to page sections (to my knowledge).

Reply:
https://stackoverflow.com/questions/15394347/adding-a-cross-reference-to-a-subheading-or-anchor-in-another-page

Seems like it is possible using autosectionlabel? It's okay for now though.

Reply:
Ah, that's neat. I'll use that in the future!